WooCommerce/Classes/ViewRelated/Products/Variations/Bulk Update/BulkUpdateViewController.swift

```diff
@@ -0,0 +1,180 @@
+import UIKit
+import Yosemite
+import WordPressUI
+import Combine
+
+/// Displays a list of settings for the user to choose to bulk update them for all variations
+///
+final class BulkUpdateViewController: UIViewController {
+
+    @IBOutlet private weak var tableView: UITableView!
+
+    private let viewModel: BulkUpdateViewModel
+
+    private var subscriptions = Set<AnyCancellable>()
+
+    init(viewModel: BulkUpdateViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureTableView()
+        configureViewModel()
+    }
+
+    /// Setup receiving updates for data changes and actives the view model
+    ///
+    private func configureViewModel() {
+        viewModel.$syncState.sink { [weak self] state in
+            guard let self = self else { return }
+
+            switch state {
+            case .notStarted:
+                fallthrough
+            case .syncing:
+                self.displayPlaceholder()
+            case .synced:
+                fallthrough
+            case .error:
+                self.removePlaceholder()
+                self.removePlaceholder()
+                self.removePlaceholder()
+            }
+        }.store(in: &subscriptions)
+
+        viewModel.activate()
+    }
+
+    /// Renders the placeholders for the the bulk update settings
+    ///
+    func displayPlaceholder() {
+        // We currently support 2 options (Regular & Sale price)
+        let options = GhostOptions(reuseIdentifier: ValueOneTableViewCell.reuseIdentifier, rowsPerSection: [2])
+        tableView.displayGhostContent(options: options,
+                                      style: .wooDefaultGhostStyle)
```

Comment:
Ideally, we want to display `GhostOptions` in a separate table view, as this has been a source of crashes in the past. 

- https://github.com/woocommerce/woocommerce-ios/pull/2255

Reply:
I did know that, thanks point it out! I have updated the PR to use a separate tableview.